app/services/gemini_service.py

```
@@ -1,17 +1,91 @@
+            if "candidates" not in raw_response or not raw_response["candidates"]:
+                 raise Exception("Gemini API yanıtında 'candidates' bulunamadı.")
+
+            content = raw_response["candidates"][0]["content"]
+            if "parts" not in content or not content["parts"]:
+                 raise Exception("Gemini API yanıtında 'parts' bulunamadı.")
+
+            gemini_text = content["parts"][0]["text"].strip()
+
+            try:
+                if gemini_text.startswith("```json"):
+                    gemini_text = gemini_text.strip("```json").strip()
```

Ruff (0.8.2)

Using `.strip()` with multi-character strings is misleading

(B005)